<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Project3</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/picture/img/myGrade.png</normaloff>:/picture/img/myGrade.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>800</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Score">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Score: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_About">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>565</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Start">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="LableGameStart">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>611</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Score_number">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarEnemyHealth">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Health">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Health:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Health_number">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/picture/img/raiden3.ico</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Health_number_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>42</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/picture/img/raiden3.ico</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Health_number_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>330</y>
      <width>57</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/picture/img/raiden3.ico</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Health_number_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>89</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/picture/img/raiden3.ico</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Health_number_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>184</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/picture/img/raiden3.ico</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Skill">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>390</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Skill:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_skill_number">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>480</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/picture/img/trump (icon).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Label_skill_number_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/picture/img/trump (icon).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Label_skill_number_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>480</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/picture/img/trump (icon).png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="button_quit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>680</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>button_Start</tabstop>
  <tabstop>button_About</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
